guard/resources/validate/functions/rules/converters.guard

```diff
@@ -0,0 +1,47 @@
+let asg = Resources.*[ Type == 'AWS::AutoScaling::AutoScalingGroup' ]
+
+
+let c = '1'
+rule test_parse_int when %asg !empty {
+    let min = parse_int(%asg.Properties.MinSize)
```

Comment:
do you think adding another condition with original datatype like `"5.0" == MinSize` will be redundant and not useful?

I think it will help understand the rule better for user with no context.

Reply:
I think its redundant due to we can see in the actual template what the rule is. This is done in the unit tests. I think this would be a good thing to add for when we update our customer facing docs / examples of how to convert them.